phoenix-core/src/it/java/org/apache/phoenix/monitoring/PhoenixLoggingMetricsIT.java

```diff
@@ -156,9 +156,9 @@ public void testPhoenixMetricsLoggedOnClose() throws Exception {
         // Autocommit is turned off by default
         // Hence mutation metrics are not expected during connection close
```

Comment:
this comment is still here though (are mutation metrics now expected during connection close?

Reply:
Thanks. Yes, now when `auto-commit` is off then we do expect mutation metrics to be there on connection close() as we are now recording metrics for `executeUpdate()` call also.

Reply:
Do you think this could be an issue?

Reply:
I don't fully understand the scenario and the impact. The test makes me think that before, without a call to `commit` or `setAutoCommit(true)`, then an `executeUpdate` is insufficient to make any writes, and therefore you have no mutation metrics. But now, even without a `commit`, we have some write metrics. But what are those metrics? Is it just the time taken for the mutation planning? Should the test confirm that those are the only metrics present?

Otherwise no, it does not seem to be an issue.

Reply:
> Should the test confirm that those are the only metrics present?

Sure, I can add this assertion also. Thanks